### DISCUSSION TITLE
Just wanna use apps

### DISCUSSION THREAD

**Discussion by thaBear80** [community/community#132868] [New to GitHub]
I don’t know anything about coding . I keep getting sent here thinking I’m getting an app or all or even ask to download. Like I was sent here looking for ton-connect. I thought it an app to connect different wallets

**Comment by ghostinhershell** [comment]
Hello @thaBear80, 

Thanks so much for reaching out in our community. Just to be clear, are you looking for a particular app hosted on GitHub called ton-connect?

**Comment by AmalFrancisOlakengil** [comment]
Are you looking for this??

https://tonhub.com/download
